### ISSUE TITLE
Text without header migrates to preamble

### ISSUE THREAD

**Issue by mrvocalplay** [orgzly/orgzly-android#238]
I really love your app and I'm using it on a daily basis for organization and for writing.

I like to Start to writing my articles on my computer and then sync them with orgzly to continue on the go. My articles do have a intro without a heading. This intro is always displayed within the preamble instead as part of the text in orgzly.

**Comment by nevenz** [comment]
> I really love your app and I'm using it on a daily basis for organization and for writing.

Thanks!

> My articles do have a intro without a heading. This intro is always displayed within the preamble instead as part of the text in orgzly.

Yes, the Org file format is:

```
Preamble text. Orgzly calls it "preface".
If can be displayed in the list of notes (in the notebook view) if you enable that option in Settings (by setting "Notebook's preface" to "Display").

* This is the first note

This is first note's content. Both are displayed in Orgzly by default in notebook view
```

So everything before the first `* Header` is displayed in preface.

What is "part of the text" you are referring to (where you'd expect intro to be displayed)?

**Comment by mrvocalplay** [comment]
As part of the text I mean part of the article which should not be in the preface, but also does not have an * headline. The prefaces does start with an
#+TITLE
For example

**Comment by nevenz** [comment]
Ah I see. "Preface" is currently everything above the first heading and it's all stored in a single view.

Yes, I think it makes sense to keep (and control the visibility of) "#+" lines separately from the text below. Especially since eventually there will be an UI for editing those "in-buffer settings".

**Comment by mrvocalplay** [comment]
Yes this would be nice to control the visibilities separately. The UI you are talking about also sounds great..
